scripts/messages.sh

```diff
@@ -349,8 +349,8 @@ function leveragelp_open() {
             "leveragelp_open": {
                 "amm_pool_id": 2,
                 "collateral_asset": "'"$usdc_denom"'",
-                "collateral_amount": "10000000",
-                "leverage": "5.0",
+                "collateral_amount": "0",
```

Comment:
should we change this to > 0 , so that script doesn't fail ?

Reply:
Changes done